modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectEntry/ModalDeleteObjectEntry.tsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayButton from '@clayui/button';
+import ClayModal from '@clayui/modal';
+import {fetch, openToast} from 'frontend-js-web';
+import React from 'react';
+
+interface ItemData {
+	actions: ItemDataDelete;
+	id: number;
+}
+
+interface ItemDataDeleteProps {
+	href: string;
+	method: string;
+}
+
+interface ModalDeleteObjectEntry {
+	closeModal: voidReturn;
+	itemData: ItemData;
+	loadData: voidReturn;
+}
+
+type ItemDataDelete = {
+	delete: ItemDataDeleteProps;
+};
+
+type voidReturn = () => void;
+
+export default function ModalDeleteObjectEntry({
+	closeModal,
+	itemData,
+	loadData,
+}: ModalDeleteObjectEntry) {
+	async function handleDelete() {
+		const deleteURL = itemData.actions.delete.href;
+
+		fetch(deleteURL.replace('{id}', String(itemData.id)), {
+			method: 'DELETE',
+		})
+			.then(({ok}) => {
+				if (ok) {
+					closeModal();
+					loadData();
+					openToast({
+						message: Liferay.Language.get(
+							'your-request-completed-successfully'
+						),
+						type: 'success',
+					});
+				}
+				else {
+					throw new Error();
+				}
+			})
+			.catch(() => {
+				openToast({
+					message: Liferay.Language.get(
+						'an-unexpected-error-occurred'
+					),
+					type: 'danger',
+				});
+			});
+	}
+
+	return (
+		<>
```

Comment:
@Artartur Please, call the ClayModal component instead of using the empty tag.

Reply:
Done!